apps/condo/domains/common/components/LayoutContext.tsx

```diff
@@ -39,7 +39,7 @@ export const LayoutContextProvider: React.FC = (props) => {
         setIsCollapsed(!isCollapsed)
     }
 
-    const isSmall = !breakpoints.DESKTOP_SMALL
+    const isSmall = !breakpoints.TABLET_LARGE
```

Comment:
It's a bit strange to continue support this API if we use breakpoints :)))
At least add a todo to rework / kill `isSmall` as a part of responsive screens task